backend/examples/user_custom_properties/default/locale/fr.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Tagline": "Slogan",
+  "Localization": "Localisation",
+  "Hobbies": "Loisir",
+  "Personal Website": "Site web personnel",
+  "Birthday": "Date d'anniversaire",
+  "Telephone": "Numéro de téléphone",
```

Comment:
`Telephone` is not `Phone` or `Phone number` ?

Reply:
Phone or Phone number indeed.
Phone is shorter.

Reply:
```suggestion
  "Phone": "Numéro de téléphone",
```